src/main/java/io/citrine/jcc/search/analysis/query/CategoricalAnalysis.java

```diff
@@ -167,7 +188,7 @@ private void readObject(ObjectInputStream in) throws IOException, ClassNotFoundE
      */
     private void readObjectNoData() throws ObjectStreamException {}
 
-    private static final long serialVersionUID = 6285820138539599732L;
+    private static final long serialVersionUID = 6285820138539599733L;
```

Comment:
Was there a corresponding change to CSL?  
Also, do we not use JSON serialisation?

Reply:
Not sure what you mean by corresponding change to CSL?  If you mean CSL version, which repos do I need to bump for this?
I sent an email asking Mark about this serializationUID, he said bump whenever you change schema.  Since JSON doesnt seem to use it, and Im not aware we serialize these objects anywhere other than web, it might not matter.